core/templates/dev/head/pages/exploration_player/PlayerTranscriptServiceSpec.js

```diff
@@ -27,31 +27,31 @@ describe('Player transcript service', function() {
   it('should reset the transcript correctly', function() {
     pts.addNewCard('First state', {
       a: 'b'
-    }, 'Content HTML', '<oppia-text-input-html></oppia-text-input-html>');
+    }, 'Content HTML', '<oppia-text-input-html></oppia-text-input-html>', null);
```

Comment:
Here and elsewhere in this file: why add the fifth arg?

Reply:
Oh, this was added when I had added the  conceptCard field. Will remove these.

Reply:
Done.